web-registry/src/pages/BasketDetails.tsx

```diff
@@ -6,12 +6,12 @@ import { BasketOverview, BasketEcocredits } from '../components/organisms';
 
 const BasketDetails: React.FC = () => {
   const { basketDenom } = useParams<{ basketDenom: string }>();
-  const { dataOverview, dataBasketBatches } = useBasketDetails(basketDenom);
+  const data = useBasketDetails(basketDenom);
```

Comment:
If the basketDenom doesn't exist, there are endless requests to the server, which is not ideal. Can we short-circuit the request loop somehow if a request fails? Otherwise the other requests keep going if a user goes to any URL other than a correct basket_denom.

I'd be willing to defer this to an error-handling story, if we make sure to document this case.

Reply:
I believe this should be addressed before we merge this

Reply:
Fixed up. Now we throw the error from the try/catch of the ledger api query functions. And in the hooks we set the error status and we take it into account in the effects that manage the requests (and it is also exported from the hook in case you want to manage it in the UI)

Reply:
Thanks. Tested and looks good now.